tasks/main.yml

```
@@ -192,7 +192,7 @@
 
   - name: Create default Beats input pipeline inputs
     template:
-      src: redis-output.conf.j2
+      src: redis-input.conf.j2
```

```suggestion
      src: redis-output.conf.j2
```

This change slipped into this PR and shouldn't be handled here.